src/pages/MainPage/components/ProceedContainer/components/SumaryContainer/components/Shipping/styles.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  border: 1px solid black;
+  padding: 10px;
+  background: ${(props) => props.theme.colors.ligthGray};
+  @media (max-width: 800px) {
```

Comment:
mozesz porobic sobie helpery do media, typu:

export const media = {
lg: '@media (max-width: 800px)'
}

i potem tutaj ${media.lg}

pisalem Ci kiedys w komentarzu breakpointy z Antd ze sa dobre, sprawdzalo mi sie do tej pory w projektach
{
  xs: '480px',
  sm: '576px',
  md: '768px',
  lg: '992px',
  xl: '1200px',
  xxl: '1600px',
}

Reply:
Racja wykorzystałem to już w wallet-example repository a tu jeszcze nie.
Done.